substrate/frame/support/src/migrations.rs

```diff
@@ -496,6 +496,14 @@ pub trait SteppedMigration {
 		None
 	}
 
+	/// This function is optional and provides information to the migration framework
+	/// to know which storage prefixes are being migrated. It can be helpful to let
+	/// chain explorers know which part of the state is possibly in a state where it
+	/// cannot be read correctly.
+	fn migrating_prefixes() -> impl IntoIterator<Item = Vec<u8>> {
```

Comment:
```suggestion
	/// Returns the prefixes of the keys to be migrated.
	///
	/// This function is optional and provides information to the migration framework
	/// to know which storage prefixes are being migrated. It can be helpful to let
	/// chain explorers know which part of the state is possibly in a state where it
	/// cannot be read correctly.
	fn migrating_prefixes() -> impl IntoIterator<Item = Vec<u8>> {
```